web/Migrations/20220803104341_AddLatLonToProperty.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BricksAndHearts.Migrations
+{
+    public partial class AddLatLonToProperty : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<decimal>(
+                name: "Lat",
+                table: "Property",
+                type: "decimal(18,2)",
```

Comment:
`DECIMAL(18, 2)` means 18 decimal digits of precision, with 16 of them to the left of the decimal point, and 2 to the right - this works well for money, but doesn't make sense for us.

Latitudes and longitudes go from 0 to 360 (or -180 to 180), so we want three digits to the left of the point. Nine decimal places is plenty so `DECIMAL(12, 9)` would work.